desktop/npm-helper.js

```diff
@@ -159,11 +163,22 @@ function postInstall () {
   }
 
   if (!process.env.KEYBASE_SKIP_DEV_TOOLS) {
-    const modules = Object.keys(postinstallGlobals).map(k => `${k}${postinstallGlobals[k]}`).join(' ')
-    exec(`npm install -g -E ${modules}`)
+    Object.keys(postinstallGlobals).forEach(k => {
+      childProcess.exec(`npm -g ls ${k}${postinstallGlobals[k]}`, {},
+        err => {
+          if (err) {
+            console.log('>>>>> Missing dev tooling ', k, postinstallGlobals[k], 'please run "npm run setup-dev-tools"')
```

Comment:
The space at the end of `'>>>>> Missing dev tooling '` is redundant since `console.log` will add a space too.